learning/step_by_step/scripting.rst

```diff
@@ -91,129 +102,126 @@ look like the image below. You can set the text in the Inspector pane.
 
 .. image:: /img/label_button_example.png
 
-Finally, save the scene, a fitting name could be "sayhello.tscn"
+Finally, save the scene, with a name such as "sayhello.tscn"
 
 .. _doc_scripting-adding_a_script:
 
 Adding a script
 ~~~~~~~~~~~~~~~
 
-Right click on the panel node, then select "Add Script" in the context
+Right click on the panel node, and then select "Add Script" in the context
 menu:
 
 .. image:: /img/add_script.png
 
-The script creation dialog will pop up. This dialog allows to select
-the language, class name, etc. GDScript does not use class names in
-script files, so that field is not editable. The script should inherit
-from "Panel" (as it is meant to extend the node, which is of Panel type,
-this is automatically filled).
+The script creation dialog will pop up. This dialog allows you to set the
```

Comment:
Tried to reorganize this section a bit to add a bit more detail about the "Name" and "Inherits" fields. Let me know what you think